### PR TITLE
Add a page highlighting OpenCost in the news

### PR DESCRIPTION
**Pull request by mattray**
Signed-off-by: Matt Ray <github@mattray.dev>

### CODE REVIEW

**Comment by mattray** [comment]
@alexkubecost ^^

**Comment by mattray** [comment]
https://capture.dropbox.com/Jdn6camgjNpD0M0t

File: src/pages/press.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: OpenCost in the News
+description: OpenStack in the News
```
Comment (dwbrown2) [inline src/pages/press.md:3]: OpenStack!

**Comment by dwbrown2** [comment]
Just going off screenshot, but is there any chance we can add simple styling to the page? For example, maybe convert links to be bullet point list?

File: src/pages/press.md (hunk 1 above)
Comment (mattray) [reply src/pages/press.md]: 🤦

**Comment by mattray** [comment]
@dwbrown2 updated to use bullets. It's Markdown rather than React, so the formatting is pretty limited, but I can try to convert it to React + Tailwind if you think it's worthwhile
Screen Shot 2022-12-14 at 6 20 38 PM

**Comment by dwbrown2** [comment]
This works for me, unless @Annalillie has other input!

**Comment by Annalillie** [comment]
This is fantastic! I have no additional input here.

**Pull request merged**